### ISSUE TITLE
Node customisation 

### ISSUE THREAD

**Issue by Adrien-Zemma** [projectstorm/react-diagrams#526]
Is it possible to customize the shape and content of a node ?
And also, where can I found true documentation about this project.

**Comment by renato-bohler** [comment]
> Is it possible to customize the shape and content of a node ?

Yes, it is possible, as shown [here](http://projectstorm.cloud/react-diagrams/?path=/story/custom-models--custom-diamond-node).

> And also, where can I found true documentation about this project.

Unfortunately, for now the best documentation seems to be some markdowns inside the repository and the code itself. I'm currently developing an app based on this library, and I've been learning mainly from the demo project and from experimenting.

**Comment by renato-bohler** [comment]
Oh, and you can join us on [Gitter](https://gitter.im/projectstorm/react-diagrams) aswell.

**Comment by kamleshgorasiya** [comment]
Hello Renato Böhler,

Do you have ideas on how to implement a group kind of node feature?
My mean of Group node is It can have multiple child nodes or groups with the same attributes set as simple node have except it can contain child node or childe group.
like given in the image

![image](https://user-images.githubusercontent.com/6419099/75142772-c50d3f00-5719-11ea-94d8-3d3ccbcc7e05.png)

**Comment by renato-bohler** [comment]
Hmm, I've seen someone asking about this on Gitter... there's a branch on this project called [`grouped_nodes`](https://github.com/projectstorm/react-diagrams/tree/grouped_nodes), but it seems inactive for over a year now.

Unfortunately, that's all I know. I have no clue about the best way to develop such a feature :(

**Comment by carldou** [comment]
Such grouping function is also important to me, hope to see that soon.

**Comment by pierre-moire** [comment]
I am renaming the ticket as it is not reflecting its purpose

**Comment by pierre-moire** [comment]
Please rename the ticket to make sure the title aligns with the purpose
